client/src/locales/translations/de.json

```diff
@@ -12,7 +12,10 @@
     "profile": "Profil",
     "addOrg": "Organisation hinzufügen",
     "logout": "Ausloggen",
-    "organisations": "Organisationen"
+    "organisations": "Organisationen",
+    "language": "Sprache",
+    "switchAccount": "Switch Account",
```

Comment:
We only need to update US English file. Other languages are translated in Lokalise based on en_US.json.